app/views/content/salaries-and-benefits.md

```diff
@@ -74,21 +74,20 @@ The support package includes:
 * funded training for mentors of early career teachers
 * funded time for mentors to support early career teachers
 
-### Teaching and learning responsibility (TLR) payments
+### Get a salary uplift for taking on extra responsibilities
 
-If you take on extra ongoing responsibilities in your role, you could get a salary uplift.
+You may be able to get 'teaching and learning responsibility' (TLR) payments if you take on extra responsibilities such as:
 
-There are 2 main ranges for these (TLR 1 and TLR 2), depending on the category your duties come under:
+* progressing the education of people beyond your assigned pupils
+* leading, developing and enhancing the teaching practice of others
+
+TLR payment come in 2 main ranges (TLR 1 and TLR 2), depending on the category your extra responsibilities come under.
```

Comment:
This left me wondering what the difference between TLR 1 and 2 is. Is it related to my expertise? Seniority? Time spent?

Reply:
Fair point Pete. I didn't think we needed to overcomplicate it, but if we were to add something:

> In order to qualify for a TLR1 (higher band) payment, teachers’ duties must include in addition “line management responsibility for a significant number of people”.  The key difference between TLR1 and TLR2 posts is, therefore, the nature of line management responsibilities.  The extent, however, of the line management responsibilities needed to qualify – for example the number of teachers and/or support staff managed – is not specified.

@EmmaFrith feel free to add a line relating to line management!

Reply:
Source: https://neu.org.uk/advice/tlr-payments-and-other-allowances

Reply:
thanks both - have added another commit